content/ecosystem/sdks/sdks.toml

```diff
@@ -673,3 +673,29 @@ featured_in = []
 description = """
 Simple shell script SDK for the Matrix Client‑Server API. Works in any POSIX-compliant shell.
 """
+
+[[sdks]]
+name = "radicale-auth-matrix"
+description = """
+Matrix authentication plugin for Radicale
+"""
+maintainer = "etke.cc"
+maturity = "Stable"
+language = "Python"
+purpose = ["auth"]
```

Comment:
I suppose this is too specific for us to introduce a new category. :thinking:

Currently, it also does not show up since we hardcode the available categories here. https://github.com/matrix-org/matrix.org/blob/75fb2d476b0274c2dabd3cc67a26e3222f2ba4d6/templates/shortcodes/sdk_deck.html#L14-L20

That said, I understand why you had the idea of putting it here, but I think the radicale plugin is a better fit for the integrations category it was initially in. Given time, I would love to expand the website more towards that kind of tooling. My personal POC for that is https://tune-your-chat.github.io/ecosystem/tune/ but I find it really difficult to design a nice system that fits everything - the amount of things is simply too diverse!